37/javadoc/index.html

```diff
@@ -0,0 +1,265 @@
+<!DOCTYPE HTML>
+<html lang="en">
+<head>
+<!-- Generated by javadoc (17) -->
+<title>Overview (kafka 3.7.0-SNAPSHOT API)</title>
```

Comment:
note that this is 3.7.0-SNAPSHOT
similar to what happened in https://github.com/apache/kafka-site/pull/541 and was later corrected in https://github.com/apache/kafka-site/pull/557

I followed all the steps to the tee, so I am not sure where the disconnect is so that it creates it with the -SNAPSHOT tag.

Reply:
ok figured out why. re-generated with the RC tag and updated the docs - 
![image](https://github.com/apache/kafka-site/assets/13639618/b64352f2-d8f9-48d4-9d20-bf9b1ba2db61)